www/_posts/2017-10-30-fetch-release.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Cordova-Fetch@1.2.1 Released!"
+date: 2017-10-30
+categories: news
+tags: release tools cordova-fetch
+---
+
+We just released some changes to `cordova-fetch`!
+
+* [cordova-fetch@1.2.1](https://www.npmjs.com/package/cordova-fetch)
+
+Release Highlights:
+* Support added for node 8.
```

Comment:
Instead of support added for node 8, we should say something like `Added node 8 support to our continuous integration tests`